how-to/laravel.md

```diff
@@ -0,0 +1,651 @@
+---
+title: How to use PGroonga with Laravel
+---
+
+# How to use PGroonga with Laravel
+In this guide we will walk through building a Laravel application from scratch with PGroonga. 
+
+# Installation
```

Comment:
Could you add one more section level to all below sections?
```suggestion
## Installation
```

Reply:
Fixed👍